ingestion/src/metadata/ingestion/source/atlas.py

```diff
@@ -217,7 +219,7 @@ def ingest_lineage(self, source_guid, name) -> Iterable[AddLineageRequest]:
                 "table"
             ]["displayText"]
 
-            fqn = f"{self.config.service_name}{FQDN_SEPARATOR}{db.name.__root__}{FQDN_SEPARATOR}{table_name}"
+            fqn = get_fqdn(self.config.service_name, db.name.__root__, table_name)
```

Comment:
Same, do not add it to sources if not required by another piece of logic.

Reply:
This applies to all appearances of FQDN in the sources just to be passed to Table